client/js/shout.js

```diff
@@ -314,6 +314,11 @@ $(function() {
 		}
 	});
 
+	socket.on("topic", function(data) {
+		// .text() escapes HTML (but not quotes)
```

Comment:
At first I wondered too why I had included that comment, but it's probably a sidenote for a case when applied text wouldn't be inside tags but inside an attribute where quotes would matter.